src/pages/transactions/[tid].tsx

```diff
@@ -0,0 +1,698 @@
+            to={data.transactions.address.to}
+            forToken={data.forToken}
+            type={data.type}
+            hex={data.hex}
+          />
+        </div>
+      </GradientCardContainer>
+    </Container>
+  );
+}
+
+interface TransactionDetailSegmentOneProp {
+  transactionId: string;
+  tokenPrice: {
+    value: string;
+    symbol: string;
+  };
+  valuePrice: string;
+  status: string;
+  blockNumber: string;
+  timestamp: string;
+  transactionFee: {
+    value: string;
+    symbol: string;
+  };
+  block: string;
+  transactionType: string;
+  nonce: string;
+  address: {
+    from: string;
+    to: string;
+    contractAddress: string;
+  };
+  type: string;
+}
+
+function TransactionDetailSegmentOne({
+  transactionId,
+  tokenPrice,
+  valuePrice,
+  status,
+  blockNumber,
+  timestamp,
+  transactionFee,
+  block,
+  transactionType,
+  nonce,
+  address,
+  type,
+}: TransactionDetailSegmentOneProp) {
+  const [isFromAddressCopied, setIsFromAddressCopied] = useState(false);
+  const [isToAddressCopied, setIsToAddressCopied] = useState(false);
+  const [isTransactionIdCopied, setIsTransationIdCopied] = useState(false);
+
+  return (
+    <div className="flex flex-col gap-y-10">
+      {/* first row */}
+      <div className="flex flex-col lg:flex-row md:flex-row gap-y-[41.5px]">
+        {/* 1st flex */}
+        <div className="flex flex-col grow gap-y-2">
+          {isTransactionIdCopied ? (
+            <div className="flex flex-row gap-x-2.5 items-center">
+              <LinkText
+                testId="transaction-id-copied"
+                label={fixedTitle.copied}
+                href={`/address/${transactionId}`}
+              />
+              <GreenTickIcon data-testid="transaction-id-copied-green-tick-icon" />
+            </div>
+          ) : (
+            <div className="flex flex-row gap-x-2.5 items-center">
+              <LinkText
+                testId="trasaction-id"
+                label={truncateTextFromMiddle(transactionId)}
+                href={`/address/${transactionId}`}
+              />
+              <FiCopy
+                data-testid="transaction-id-copy-icon"
+                onClick={() =>
+                  onCopyAddressIconClick(setIsTransationIdCopied, transactionId)
+                }
+                className="text-white-50"
+              />
+            </div>
+          )}
+
+          <NumericFormat
+            data-testid="transaction-token-price"
+            className="text-white-50 lg:text-[32px] md:text-2xl text-xl"
+            value={
+              type === "tokenized" || type === "contract"
+                ? tokenPrice.value
+                : "0"
+            }
+            decimalScale={type === "tokenized" || type === "contract" ? 8 : 2}
+            suffix={` ${tokenPrice.symbol}`}
+          />
+
+          <NumericFormat
+            data-testid="transaction-value-price"
+            className="text-white-700"
+            thousandSeparator
```

Comment:
margin top should be 4px for desktop

Reply:
added mb padding

<img width="1011" alt="image" src="https://user-images.githubusercontent.com/57183851/202429995-baa7828e-7edc-4b4a-b7e6-294d577ea03e.png">

<img width="875" alt="image" src="https://user-images.githubusercontent.com/57183851/202430049-fa83e2ba-4757-4432-af54-13c2ee5ee1d9.png">